### PR TITLE
Final awesome books application

### PR DESCRIPTION
**Pull request by tajemouti**
1. Created a new branch for this feature.
2. Added Navigation Bar to Main page.
3. Refactored the HTML page.
4. Added a Contact section and a footer element to the page.
5. Implemented today's date to the page.
6. Added an event listener to navigation bar list.
7. Fixed linters errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,18 +8,50 @@
...
+        <ul class="contact-list">
+            <li>Our e-mail: mail@mail.com</li>
+            <li>Our phone number: 00212586534422</li>
+            <li>Our address: Streetname 22, 84503 City, Country</li>
+        </ul>
+    </section>
+
+    <footer>
+        Copyright ...
+    </footer>
```
Comment (muneebulrehman) [inline index.html:53]: As per the wireframe provided, the footer is at the bottom of the page. However, in your case, it is floating in the middle of the page.  Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job adding the navigation.
- Contribution by both partners.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KDlamini) [reply index.html:53]: Well resolved ✔️ 

image

**Review by KDlamini — APPROVED** [review]
### Hi team @tajemouti, @MozamelJawad  🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Professional pull request
✔️ Three pages - Books list, Add book form, and Contact info are present
✔️ A navigation bar allows for navigating the three pages

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**